### DISCUSSION TITLE
Capstone Flask UI

### DISCUSSION THREAD

**Discussion by Kukuhws** [teamalgoritma/community#204] [Q&A] [answered]
Dear Mentor

Mohon bantuan konfirmasi nya, apakah untuk task nomor 6 ini memang harus sama dengan bawahnya persis dengan posisi masing-masing kolomnya? dan jika memang iya apakah ada insight kenapa punya saya antara kolom Rating & Reviews nya tertukar? 

![image](https://user-images.githubusercontent.com/107305755/178405504-9de8f64e-14e9-41da-8f59-c942d985f262.png)

**Comment by amaliasf** [comment, marked as answer]
Halo Pak @Kukuhws 
Berdasarkan output yang dihasilkan seharusnya sudah tepat, hanya saja terjadi perbedaan posisi kolom. 
Hal ini kemungkinan terjadi karena ketika agregrasi Bapak menyebutkan kolom Rating terlebih dahulu, baru kolom Reviews.
Untuk menyamakannya, Bapak hanya perlu menukarkan urutan penyebutan kolom pada kodingan Bapak.

Semoga membantu :)
